src/m1_facilitator/repositories/TransactionRepository.ts

```diff
@@ -25,7 +25,7 @@ import Utils from '../../common/Utils';
 /**
  * An interface, that represents an each row of TransactionRepository.
  */
-class TransactionModel extends Model {
+class TransactionModel {
```

Comment:
We can proably remove this class?

Reply:
yes, we can remove it.
I would suggest to make `TransactionModelInterface` otherwise we would have to use `any` in `convertToTransaction` transaction method of `TransactionRepository`